mentors/vimscript/y.json

```diff
@@ -4,7 +4,7 @@
     "name": "Josh Goebel",
     "link_text": "Blog: Running Blind",
     "link_url": "http://runningblind.com/",
-    "avatar_url": "",
-    "bio" :" I mostly write Ruby on Rails professionally but for fun (and possibly profit one day) I dabble with Elixir, Nim, Arduino, and such. I’m really digging both Nim and Vim lately."
+    "avatar_url": null,
+    "bio": "I mostly write Ruby on Rails professionally but for fun (and possibly profit one day) I dabble with Elixir, Nim, Arduino, and such. I’m really digging both Nim and Vim lately."
   }
```

Comment:
Could you please revert this? I'd prefer if @yyyc514 would do the change himself to make sure he does actually want this change to happen.

Reply:
What is the effective diff here between `null` and `""` anyways from the UI perpective?

The extra space is just a typo, lol.  I thought I looked around at the time and saw other people using blank strings, but I can't recall now.  I've slept since then.

Reply:
One could argue that with an empty string it should show no image at all, but it tries anyway.

When set to `null` instead, it infers the GitHub avatar.

Currently it looks like this: https://exercism.io/tracks/vimscript/mentors

Reply:
The default GitHub avatar would be better than looking broken.

Reply:
From the `README`:

>  If you want to leave a field blank, set it to an empty string.

I do not see why anything should look broken. There simply should not be an avatar. Maybe it looks different than other cards, but it shouldn't look broken. If it does, we need to fix the website and not other peoples profile.

Reply:
I agree.  Here is a snippet of what I see.
![image](https://user-images.githubusercontent.com/6978003/52442526-5ac73380-2af1-11e9-9a35-4c888ef22489.png)

I think the site might need to be updated to so that "white" space is placed where the avatar would normally be located....